Design: 검사페이지 소개/질문 섹션 반응형 구현

H-JuKyung opened the pull request:
## 📝 변경 사항

> 커밋 단위로 명시적으로 짧게 작성합니다.

1. 직업 검사 페이지 소개와 질문 섹션에 반응형 구현

## 🔍 변경 사항 세부 설명

> 세부 설명을 작성합니다.

X

## 🕵️‍♀️ 요청사항

> 팀원들에게 테스트나 리뷰를 요청합니다.

- 반응형 확인해보시고 수정해야 할 사항 있으면 수정하셔도 좋습니다.
- 피드백 주셔도 좋습니다.

## 📷 스크린샷 (선택)

> UI 변경이 있는 경우 스크린샷이나 GIF를 첨부합니다.

https://github.com/user-attachments/assets/4460daf1-a3ac-4e1d-a6b6-d40ee1f20520

## ✅ 작성자 체크리스트

- [x] Self-review: commit 이나 오타 없이 코드가 스스로 검토됨
- [x] 로컬에서 모든 기능이 정상 작동함(버그수정 /기능에 대한 테스트)
- [x] 린터 및 포맷터로 코드 정리됨

H-JuKyung committed:
Design: TestInformSection 반응형 구현

H-JuKyung committed:
Design: Card 컴포넌트 반응형 구현

H-JuKyung committed:
Merge branch 'Design-검사페이지-소개-질문-반응형' of https://github.com/hyonun321/JobThoughts into Design-검사페이지-소개-질문-반응형

H-JuKyung committed:
Design: 직업 검사 페이지 모두 반응형 적용 완료

H-JuKyung committed:
Merge branch 'Design-검사페이지-소개-질문-반응형' of https://github.com/hyonun321/JobThoughts into Design-검사페이지-소개-질문-반응형

sejinbaek commented:
![image](https://github.com/user-attachments/assets/16107a52-a846-445c-9a44-e2c768740222)
저만 그런 건지 모르겠지만 F12에서 말고 그냥 화면 사이즈를 줄이면 이렇게 겹쳐보입니다 ! 다른 분들도 그러신가요?

yeom-kenco commented:
> ![image](https://private-user-images.githubusercontent.com/117803382/445117294-16107a52-a846-445c-9a44-e2c768740222.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDc2NTU1NTcsIm5iZiI6MTc0NzY1NTI1NywicGF0aCI6Ii8xMTc4MDMzODIvNDQ1MTE3Mjk0LTE2MTA3YTUyLWE4NDYtNDQ1Yy05YTQ0LWUyYzc2ODc0MDIyMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNTE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDUxOVQxMTQ3MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hZGRiY2E3ZTUwM2UwODRlYTk3NjEyMWQwOGRhZTNjNTBmZTgyZDUzNDJlYjcyNDFhOTJkMzQwZjZkMWE0YmE4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.HTSJw01B2mR_KX7Yg587CpUiSpeuSauPL2AZTCvbUMU) 저만 그런 건지 모르겠지만 F12에서 말고 그냥 화면 사이즈를 줄이면 이렇게 겹쳐보입니다 ! 다른 분들도 그러신가요?

저도 가로까지 줄이면 그렇게 보이기는합니다! 그냥 세로만 줄이면 예쁘게 보여요.... 근데 유저가 가로세로를 저렇게까지 줄여서 할까요??.....그렇게까지 줄여서 할거면 쓰지말아라!!!!!는 너무 무책임한 프론트엔드개발자에요?ㅜㅜ

yeom-kenco committed:
Design: 배경 아이콘 레이아웃 반응형으로 재배치

yeom-kenco commented:
배경에 떠다니는 아이콘들 위치가 반응형으로 변경될때 조금 어색한거같아서, 여기 브랜치에 커밋으로 디자인 수정했습니다~
![Animation8](https://github.com/user-attachments/assets/03268428-41dc-4dee-9b0e-3cad556e2a7c)

sejinbaek commented:
> 저도 가로까지 줄이면 그렇게 보이기는합니다! 그냥 세로만 줄이면 예쁘게 보여요.... 근데 유저가 가로세로를 저렇게까지 줄여서 할까요??.....그렇게까지 줄여서 할거면 쓰지말아라!!!!!는 너무 무책임한 프론트엔드개발자에요?ㅜㅜ

그거 제가 한번 해보겠습니다.... 도저언 !!

hyonun321 committed:
Merge branch 'main' into Design-검사페이지-소개-질문-반응형

sejinbaek committed:
Refactor: Test 페이지 반응형 처리 보완

- 의미없는 요소 및 적용되지 않는 스타일 제거
- 카드 사이즈, 폰트 사이즈, margin, padding을 clamp()방식으로 조절해 및 화면 축소 시 위아래가 잘리는 문제 해결

sejinbaek committed:
Chore: 스타일 가이드라인 제거 및 미사용 스타일 컴포넌트 제거

sejinbaek commented:
![Animation2](https://github.com/user-attachments/assets/76513141-f304-4948-9e61-d5a7de7f1cc2)
![Animation1](https://github.com/user-attachments/assets/b7874a7f-4b79-495f-b65c-6d032ce0953d)
반응형 처리 완료하고 스타일이 적용되지 않는 코드 정리했습니당~!

H-JuKyung commented:
다들 감사합니다ㅜㅜ👍🏻👍🏻👍🏻

sejinbaek reviewed (approved):
여기 브랜치 활활 타오릅니다🔥 모두 수고하셨습니당 !

sejinbaek committed:
Merge branch 'main' into Design-검사페이지-소개-질문-반응형

hyonun321 reviewed (approved):
고생하셨습니다 !!!
